composer/trainer/ddp.py

```diff
@@ -1,5 +1,7 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
```

Comment:
There's a typo at the bottom -- "contextmanager" should be "context manager"

Reply:
Also, even if the module remains public, ddp_sync_context and prepare_ddp_module can be private functions. Not a big deal -- could go either way -- but could simplify the docs a bit.